.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "window.zoomLevel": 5,
```

Comment:
This line modifies the other ppl editors, please left the eslint/prettier related configs